src/components/ReportActionItem/ReportPreview.js

```diff
@@ -95,11 +94,10 @@ const defaultProps = {
 };
 
 function ReportPreview(props) {
-    const reportAmount = CurrencyUtils.convertToDisplayString(ReportUtils.getMoneyRequestTotal(props.iouReport), props.iouReport.currency);
     const managerEmail = props.iouReport.managerEmail || '';
-    const managerName = ReportUtils.isPolicyExpenseChat(props.chatReport) ? ReportUtils.getPolicyName(props.chatReport) : ReportUtils.getDisplayNameForParticipant(managerEmail, true);
     const isCurrentUserManager = managerEmail === lodashGet(props.session, 'email', null);
     const bankAccountRoute = ReportUtils.getBankAccountRoute(props.chatReport);
+    const displayingMessage = ReportUtils.getMoneyRequestReportActionMessage(props.iouReport, props.action);
     return (
         <View style={[styles.chatItemMessage]}>
             {_.map(props.action.message, (message, index) => (
```

Comment:
The message is not used here anymore
```suggestion
            {_.map(props.action.message, (message, index) => (
```

Reply:
@hoangzinh Yep lets try removing it